transfer/message/shape/transfer-request-message-shape.ttl

```diff
@@ -62,11 +64,11 @@ shapes:TransferRequestMessageShape
 	sh:property [
 		a sh:PropertyShape ;
 		sh:path dspace:callbackAddress ;
-		sh:datatype xsd:anyURI ;
+		sh:datatype xsd:string ;
```

Comment:
I am pretty sure that the JSON Context actually maps the value object to xsd:anyURI, therefore, the old datatype was correct.